extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtension.scala

```diff
@@ -40,7 +40,7 @@ class RangerSparkExtension extends (SparkSessionExtensions => Unit) {
 
   override def apply(v1: SparkSessionExtensions): Unit = {
     v1.injectResolutionRule(_ => new RuleReplaceShowObjectCommands())
-    v1.injectResolutionRule(new RuleApplyRowFilterAndDataMasking(_))
+    v1.injectPostHocResolutionRule(new RuleApplyRowFilterAndDataMasking(_))
```

Comment:
instead of changing L43, you can try L44, `v1.injectPostHocResolutionRule(_ => new RuleEliminateMarker())`

Reply:
It don't work. The exception is the same.